Private/Get-JenkinsUserInfo.ps1

```diff
@@ -0,0 +1,23 @@
+function Get-JenkinsUserInfo {
+    [CmdletBinding()]
+    param (
+        [String] $Username = $script:apiUsername,
+        [SecureString] $Password = $script:apiPassword,
+        [String] $TargetUsername = "me"
```

Comment:
Great input. Thanks and updated.

Reply:
However, making the parameter mandatory defeats the default value when empty behavior.
`[ValidateNotNullOrEmpty()]` is the right implementation instead of mandatory.